src/common/integrations/GoogleDrive.js

```diff
@@ -57,18 +57,20 @@ class GoogleDriveUtil {
                     })
                     .then(
                         () => {
-                            this.auth = gapi.auth2.getAuthInstance();
-                            this.auth.isSignedIn.listen(isLoggedIn => this.updateLoginStatus(isLoggedIn));
-                            this.updateLoginStatus(this.auth.isSignedIn.get());
-                            globalObserver.emit('googledrive.initialized', true);
-
-                            $('#integrations').removeClass('invisible');
-                            $('#save-google-drive')
-                                .parent()
-                                .removeClass('invisible');
-                            $('#load-google-drive')
-                                .parent()
-                                .removeClass('invisible');
+                            this.auth = gapi?.auth2?.getAuthInstance?.();
+                            if (this?.auth) {
```

Comment:
If `this.auth` has already been assigned in the upper line,
 do we still need the `?` marker for `this.?auth`

Reply:
yes i will remove the question its not required 👍